apps/files/src/models/Tab.js

```diff
@@ -52,6 +59,10 @@ export default class Tab {
 		return this.#component
 	}
 
+	get enabled() {
```

Comment:
```suggestion
	get isEnabled() {
```

to be consistent with `isLegacyTab`